tests/Ethereum/TWEthereumAbiTests.cpp

```diff
@@ -51,6 +51,29 @@ TEST(TWEthereumAbi, FuncCreate1) {
     TWEthereumAbiFunctionDelete(func);
 }
 
+TEST(TWEthereumAbi, FuncCreateSkip) {
```

Comment:
I don't quite understand what this is testing, can you please add a comment? 🙏

Reply:
Done.